### ISSUE TITLE
Training

### ISSUE THREAD

**Issue by ymliu-felix** [shaohao011/MedCCO#2]
This is a great job. I have a question, are VQA-RAD, SLAKE, and Path VQA datasets trained together, or are they trained and tested separately? If it is the former, is the Qwen2.5-VL-7B (SFT) reported in the paper also using this setting?

**Comment by shaohao011** [comment]
the former, yes, SFT also uses this setting.







-----原始邮件-----
发件人:"Yameng Liu" ***@***.***>
发送时间:2025-07-01 09:40:14 (星期二)
收件人: shaohao011/MedCCO ***@***.***>
抄送: Subscribed ***@***.***>
主题: [shaohao011/MedCCO] Training (Issue #2)


ymliu529 created an issue (shaohao011/MedCCO#2)

This is a great job. I have a question, are VQA-RAD, SLAKE, and Path VQA datasets trained together, or are they trained and tested separately? If it is the former, is the Qwen2.5-VL-7B (SFT) reported in the paper also using this setting?

**Comment by ymliu-felix** [comment]
Thank you for your reply. Also, I would like to know how to obtain open-ended results. Is it enough to remove the options from the prompt (return f "{da ['question ']} \ n {option_str}") in eval_vllm_verl_models.py?

**Comment by shaohao011** [comment]
> Thank you for your reply. Also, I would like to know how to obtain open-ended results. Is it enough to remove the options from the prompt (return f "{da ['question ']} \ n {option_str}") in eval_vllm_verl_models.py?

we'll update the corresponding codes recently, stay tuned, thanks!

**Comment by ymliu-felix** [comment]
Recently, I have been trying to reproduce the experimental results under open-ended conditions, but I have encountered some problems and would like to ask for your advice.
1. Does open-ended metrics (combined BLEU1, ROUGE1, and BERTScore.) mean calculating the average of the three for each sample?
2. Which model is used by BERTScore for score calculation? I found that the experimental results of different BERT models vary greatly.
3. I noticed that the results under closed ended conditions like LLaVA-Med (51.4) are much lower than those in the original paper (84.19). Do you know why this is?
Thank you for your reply.

**Comment by shaohao011** [comment]
> Recently, I have been trying to reproduce the experimental results under open-ended conditions, but I have encountered some problems and would like to ask for your advice.
> 
> 1. Does open-ended metrics (combined BLEU1, ROUGE1, and BERTScore.) mean calculating the average of the three for each sample?
> 2. Which model is used by BERTScore for score calculation? I found that the experimental results of different BERT models vary greatly.
> 3. I noticed that the results under closed ended conditions like LLaVA-Med (51.4) are much lower than those in the original paper (84.19). Do you know why this is?
>    Thank you for your reply.

1. open-ended metric is the same with "Open-ended Rewards" and the same \lambda

2.  we use "bert-base-uncased" which you can find at fsdp_workers.py. Yes, different BERT models may yield varying results due to the inherent instability of NLP-based metrics and the overall brevity of responses. To address this issue, we are considering using LLM as judge to ensure more reliable judgments. Besides, our intention is using open-ended VQA RL to improve close-ended performance, so you can pay more attention on this.

3. We use the benchmark setting of HuatuoGPT-Vision, and they use LLava-med without finetuning to test model's zero shot performance, you can find the similar results in LLaVa-Med paper. below are results from LLaVA-Med

<img width="518" height="462" alt="Image" src="https://github.com/user-attachments/assets/66cc1ee4-f2ff-45e8-9455-58be6c03bdaf" />
